.github/workflows/deploy.yml

```
@@ -0,0 +1,51 @@
+name: Deploy
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    # Clona o repositório.
+    - uses: actions/checkout@v4
+
+    # Instala PHP e Composer.
+    - name: Configura PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: 8.x
+        tools: composer:v2
+
+    # Valida o arquivo composer.json.
+    - name: Valida o arquivo composer.json e o arquivo composer.lock
+      run: composer validate --strict
+
+    # Armazena as dependências em cache para que não seja necessário
+    # fazer o download a cada nova execução.
+    - name: Armazena pacotes Composer
+      id: composer-cache
+      uses: actions/cache@v2
+      with:
+        path: vendor
+        key: ${{ runner.os }}-php-${{ hashFiles('**/composer.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-php-
+
+    # Instala as dependências.
+    - name: Instala as dependências
+      run: composer install --prefer-dist --no-progress
+
+    # Faz o build do site na pasta temporária output_prod.
```

O nome da pasta mudou, mas de repente é melhor remover do comentário em vez de corrigir.

```suggestion
    # Faz o build do site na pasta temporária.
```